verify.py

```diff
@@ -203,7 +203,9 @@ def __init__(self, inputpath, config, logger):
             else:
                 self.type = 'rdf'
                 self.destpath = quote((config.dir + self.relpath + config.ext))
-                self.graph = Graph().parse(self.origpath)
+                response = requests.get(self.origpath, auth=config.auth)
+                if response.statuscode == 200:
```

Comment:
Actually, never mind - I know that @jwestgard is going to refactor this class, so things will probably change overall.